spec/unit/resources/xcode_spec.rb

```diff
@@ -16,8 +16,6 @@
       .and_return(["Software Update Tool\n",
                    "\n", "Finding available software\n",
                    "Software Update found the following new or updated software:\n",
-                   "   * Command Line Tools (macOS El Capitan version 10.11) for Xcode-8.2\n",
-                   "\tCommand Line Tools (macOS El Capitan version 10.11) for Xcode (8.2), 150374K [recommended]\n",
                    "   * Command Line Tools (macOS High Sierra version 10.13) for Xcode-10.0\n",
```

Comment:
Does Sierra not need to be in the xcode spec tests?

Reply:
Technically this is just mock data so it should't matter - in fact we can leave El Cap in there as well. It's only present to test how well the resource parses the Apple SWU catalog output.